docs/daemon/cli/server.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 2
+sidebar_label: "Server"
+id: daemon-cmdline-server
+title: Server
+description: Kwild node Command Line
+slug: /daemon/cmdline/server
+---
+
+Kwild server commands provides a way to interact and operate kwild server.
+```
+kwild server --help
+
+Commands to run Kwild Server
+
+Usage:
+  kwild server [command]
+
+Available Commands:
+  start       kwil grpc server
+
+Flags:
+  -h, --help   help for server
+
+Global Flags:
+      --config string   kwild config file (.toml)
+
+Use "kwild server [command] --help" for more information about a command.
+```
+
+
+
+To start the Kwild server:
+`kwild server start --config "config-file-path"`
```

Comment:
Create new-line code blocks for all code.  Instead of:

To start the Kwild server:
`kwild server start --config "config-file-path"`

do

To start the `kwild` server:
```bash
kwild server start --config "path/to/my/config.toml"
```